gravitee-apim-console-webui/src/management/home/home-overview/home-overview.component.html

```diff
@@ -16,6 +16,9 @@
 
 -->
 <div class="home-overview">
+  <div class="home-overview__time-selector">
+    <gio-quick-time-range [formControl]="timeRangeControl" (onRefreshClicked)="refresh()"></gio-quick-time-range>
```

Comment:
I wonder if we shouldn't have the same display

here the idea is to give a little information to the user on what he sees
![image](https://user-images.githubusercontent.com/4974420/228864023-fdf92ec0-145f-4169-bad6-558ece221b1b.png)

maybe see it with Audrey at the end

Reply:
Checked with Audrey and added a new commit to define label for both tabs in the quick time range component